<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>951</width>
    <height>536</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <pointsize>9</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>Клавиатурный тренажёр</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>../../../Desktop/08293126042023_b68857f12d5374604d572de16b1ad8f2f25718e3.png</normaloff>../../../Desktop/08293126042023_b68857f12d5374604d572de16b1ad8f2f25718e3.png</iconset>
  </property>
  <property name="locale">
   <locale language="Russian" country="Belarus"/>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTextEdit" name="textEdit">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>10</y>
      <width>911</width>
      <height>311</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Consolas,monospace</family>
      <pointsize>-1</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QTextEdit {
	font-family: Consolas, monospace;
	font-size: 24px;
    border: 1px solid #dcdcdc;
}

QScrollBar:vertical {
    width: 15px; /* Ширина ползунка */
}

QScrollBar::handle:vertical {
    background: #4d90fe; /* Цвет ползунка */
    min-height: 25px; /* Минимальная высота ползунка */
}

QScrollBar::add-line:vertical,
QScrollBar::sub-line:vertical {
    border: none; /* Убираем стрелки у концов ползунка */
}

QScrollBar::add-page:vertical,
QScrollBar::sub-page:vertical {
    background: none; /* Убираем цвет фона у страницы ползунка */
}</string>
    </property>
   </widget>
   <widget class="QPushButton" name="randomText">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>340</y>
      <width>291</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>18</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string>Случайный текст</string>
    </property>
    <property name="checkable">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="startEndButton">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>420</y>
      <width>291</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>18</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string>Начать</string>
    </property>
    <property name="checkable">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QComboBox" name="textTopic">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>340</y>
      <width>291</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>18</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
   </widget>
   <widget class="QComboBox" name="textComplexity">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>420</y>
      <width>291</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>18</pointsize>
     </font>
    </property>
   </widget>
   <widget class="QLabel" name="timeLabel">
    <property name="geometry">
     <rect>
      <x>660</x>
      <y>340</y>
      <width>191</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>18</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Время: 0:00</string>
    </property>
   </widget>
   <widget class="QLabel" name="speedLabel">
    <property name="geometry">
     <rect>
      <x>660</x>
      <y>390</y>
      <width>281</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>18</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Скорость: 0 зн/мин</string>
    </property>
   </widget>
   <widget class="QLabel" name="correctnessLabel">
    <property name="geometry">
     <rect>
      <x>660</x>
      <y>440</y>
      <width>291</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>18</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Правильность: 100%</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>951</width>
     <height>36</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>13</pointsize>
    </font>
   </property>
   <widget class="QMenu" name="menu">
    <property name="font">
     <font>
      <pointsize>13</pointsize>
     </font>
    </property>
    <property name="title">
     <string>Сведения</string>
    </property>
    <addaction name="showProgramDetails"/>
    <addaction name="showAuthor"/>
   </widget>
   <widget class="QMenu" name="menu_2">
    <property name="title">
     <string>Программа</string>
    </property>
    <addaction name="resultsAction"/>
    <addaction name="separator"/>
    <addaction name="exitAction"/>
   </widget>
   <addaction name="menu_2"/>
   <addaction name="menu"/>
  </widget>
  <action name="showAuthor">
   <property name="text">
    <string>Об авторе</string>
   </property>
   <property name="font">
    <font>
     <pointsize>13</pointsize>
    </font>
   </property>
  </action>
  <action name="showProgramDetails">
   <property name="text">
    <string>О программе</string>
   </property>
   <property name="font">
    <font>
     <pointsize>13</pointsize>
    </font>
   </property>
  </action>
  <action name="resultsAction">
   <property name="text">
    <string>Результаты</string>
   </property>
   <property name="font">
    <font>
     <pointsize>13</pointsize>
    </font>
   </property>
  </action>
  <action name="exitAction">
   <property name="text">
    <string>Выход</string>
   </property>
   <property name="font">
    <font>
     <pointsize>13</pointsize>
    </font>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
